autonomy/autonomy/urls.py

```diff
@@ -18,5 +18,8 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('accounts/', include('django.contrib.auth.urls')),
+    # Accounts app
+    path('signup/', include('accounts.urls')),
```

Comment:
I think this should be accounts/signup

Reply:
Actually like this:
`
path('accounts/', include('django.contrib.auth.urls')),
    path('accounts/', include('accounts.urls')),
`

Reply:
Looks good!